Lib9c/Action/InitializeStates.cs

```diff
@@ -149,11 +149,15 @@ public override IAccountStateDelta Execute(IActionContext context)
                 .SetState(ShopState.Address, Shop)
                 .SetState(GameConfigState.Address, GameConfig)
                 .SetState(RedeemCodeState.Address, RedeemCode)
-                .SetState(AdminState.Address, AdminAddress)
                 .SetState(ActivatedAccountsState.Address, ActivatedAccounts)
                 .SetState(GoldCurrencyState.Address, GoldCurrency)
                 .SetState(Addresses.GoldDistribution, GoldDistributions);
 
+            if (AdminAddressState != null)
```

Comment:
I recommend using the `is` operator keyword to compare with the `null` value.

It's because of the below part. ([source](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/type-testing-and-cast#is-operator))

> The is operator doesn't consider user-defined conversions.

Reply:
Related to this, I originally wrote a code like `if (AdminAddressState is not null)` but Rider says I have to update C# language version.
So I changed it to use `!=` operator but I should change that to `if (!(AdminAddressState is null))`.

p.s. Should we update C# language version to use latest features?

Reply:
> Should we update C# language version to use latest features?

Yes, we should update C# language version if you want. But the lib9c must guarantee netstandard2.0 so it should check if it doesn't have any side effects.

Reply:
It seems okay when I looked [this document](https://docs.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-1-0#when-to-target-net50-or-net60-vs-netstandard). We can check it by replacing the language version with 9 or 10, and building it.

Reply:
About updating C# language version, continue at #1295